lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -22,15 +22,23 @@ import Spinner from './Spinner';
 
 /**
  * @typedef {import('./LMSGrader').User} User
+ * @typedef {import('../services/client-rpc').ClientRpc} ClientRpc
+ */
+
+/**
+ * @typedef BasicLtiLaunchAppProps
+ * @prop {ClientRpc} clientRpc
```

Comment:
It would be handy to comment, somewhere in this module, a context for what `clientRpc` is for here. It's feeling just a little abstract.

Reply:
I've added a comment to this prop here and in `LMSGrader`